.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Container to Packages
+
+on:
+  release:
+    types: [published]
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
As I understand it we should use Nexus as our container registry across the board now, so the registry for this should be `docker-oss.nexus.famedly.de`. See [this notion page](https://www.notion.so/famedly/Nexus-Registry-55dfc4abe7ef4a8ea03946d655238b5d) for more info. I will see if we can make a post in the announcements room about this to make sure everyone is made aware.

You can authenticate to `docker{-oss}.nexus.famedly.de` with the user `famedly-ci` and password stored in `secrets.REGISTRY_PASSWORD`.

Reply:
Thank you, I have changed this now